frontend/web/components/modals/CreateFlag.js

```diff
@@ -1335,11 +1341,24 @@ const CreateFlag = class extends Component {
                                                       invalid ||
                                                       !savePermission
                                                     }
+                                                    dropdownItems={[
+                                                      {
+                                                        label: releasePipelineId
```

Comment:
Do you think we could add a loader here the time to fetch the pipelines if not too much of a hustle?
My local is slow sometimes it takes up to 5sec so it emphasizes it. Leading to have Add features pipelines and as I still don't have them, my dropdown in the modal is also empty.
https://www.loom.com/share/978e0eb7ddb64146bdac10894341123b?focus_title=1&muted=1&from_recorder=1